requirements.txt

```diff
@@ -0,0 +1,6 @@
+#
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
I know it's an empty file but we should probably standardise our development environment. All the `requirements*.txt` files should be generated using a single version of python - probably 3.10 is the best option. 

Again, it doesn't really matter as long as our tests are running against all of the versions of python we want to be compatible with but just for aesthetics it would be good to be consistent here.

Reply:
Updated to version 3.10